frontend/www/js/omegaup/components/common/Requests.vue

```diff
@@ -48,10 +48,27 @@
           <td v-if="!request.accepted" class="text-center">
             <button
               class="close float-none text-danger mx-2"
-              @click="$emit('deny-request', { username: request.username })"
+              @click="showFeedbackModal = !showFeedbackModal"
             >
               ×
             </button>
+            <b-modal
+              v-model="showFeedbackModal"
+              :title="T.submitFeedbackRequireConfirmation"
+              :ok-title="T.submitFeedbackSubmit"
+              ok-variant="success"
+              :cancel-title="T.submitFeedbackCancel"
+              cancel-variant="danger"
+              @ok="
+                $emit('deny-request', request.username, resolutionText);
+                resolutionText = '';
+              "
```

Comment:
In general, if we need to perform more than one action in response to events, it's preferable to create a function like this:
```suggestion
              @ok="onDenyRequest(request.username, resolutionText)"
```
Here, we can include all the necessary code.

The function would look like this:
```ts
  onDenyRequest(username: string, resolutionText: null | string): void {
    this.$emit('deny-request', { username, resolutionText });
    this.resolutionText = null;
    this.toggleFeedbackModal(username);
  }
```
An important aspect of this function is that we aim to emit only one parameter when triggering an action. Using object destructuring syntax makes it easier to understand the expected input for the functions. Additionally, if more properties are added to the parameter object in the future, they can simply be included in the destructuring without the need to modify the function body.